nginx.tmpl

```diff
@@ -339,16 +363,41 @@ server {
 	include /etc/nginx/vhost.d/default;
 	{{ end }}
 
-	location / {
+	{{ range $container := $containers }}
+	{{/* Determine whether this container has specified a virtual path, or default to the / pattern */}}
+	{{ $location := coalesce $container.Env.VIRTUAL_PATH "/" }}
+
+	{{ $is_regexp := hasPrefix "^" $location }}
+	{{ $modifier := when $is_regexp "~" "" }}
+
+	{{ if (eq $container.Env.VIRTUAL_PATH_STRIP "TRUE") }}
```

Comment:
this call is erroring when VIRTUAL_PATH_STRIP is not set btw.

Reply:
I think it's fixed. Also, I changed `TRUE` to `true` so it's the same as in other conditions.

Reply:
awesome thanks :). I'm not too hopeful this PR will make it into master (just because previous attempts have not gotten feedback from the maintainer -) but can confirm i'm using this in my local version of the nginx-proxy and it is working well .